includes/admin/class-wc-stripe-payment-gateways-controller.php

```diff
@@ -0,0 +1,80 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
+/**
+ * Class that handles various admin tasks.
+ *
+ * @since 5.6.0
+ */
+class WC_Stripe_Payment_Gateways_Controller {
+	/**
+	 * Constructor
+	 *
+	 * @since 5.6.0
+	 */
+	public function __construct() {
+		// If UPE is enabled and there are enabled payment methods, we need to load the disable Stripe confirmation modal.
+		$enabled_upe_payment_methods = get_option( 'woocommerce_stripe_settings', [] )['upe_checkout_experience_accepted_payments'];
+		$upe_payment_requests_enabled = 'yes' === get_option( 'woocommerce_stripe_settings', [] )['payment_request'];
```

Comment:
(minor)
```suggestion
        $stripe_settings = get_option( 'woocommerce_stripe_settings', [] );
		$enabled_upe_payment_methods = $stripe_settings['upe_checkout_experience_accepted_payments'];
		$upe_payment_requests_enabled = 'yes' === $stripe_settings['payment_request'];
```

But it's probably going to need some linting :D

Reply:
Fixed!